host/3.0/buster/amd64/python/python37-context/start.sh

```diff
@@ -1,15 +0,0 @@
-#! /bin/bash
-
-# Directory name for start.sh
-DIR="$(dirname $0)"
-
-CUSTOM_PACKAGES_PY37="$HOME/site/wwwroot/.python_packages/lib/python3.7/site-packages"
-CUSTOM_VENV_PACKAGES_PY37="$HOME/site/wwwroot/worker_venv/lib/python3.7/site-packages"
-
-export PYTHONPATH=$CUSTOM_PACKAGES_PY37:$CUSTOM_VENV_PACKAGES_PY37:$PYTHONPATH
```

Comment:
This is no longer needed?

Reply:
Yes, this is same as host/2.0/stretch.
Because we move the start code from start.sh to python worker